packages/core/onLongPress/directive.test.ts

```diff
@@ -47,13 +46,6 @@ describe('vOnLongPress', () => {
     it('should be defined', () => {
       expect(wrapper).toBeDefined()
     })
-
-    it('should trigger longpress after 500ms', async() => {
```

Comment:
It was removed because `@vue/test-utils` only support `Vue3`, which will never trigger vue2's directive `inserted` hook. 

So maybe we just need to test it manually to make sure it works .

Reply:
Could wrap it with `isVue3` from `vue-demi`

Reply:
Ok, done.